infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/environment/OpenShiftEnvironmentParser.java

```diff
@@ -153,6 +162,19 @@ private void normalizeEnvironment(
     return labels;
   }
 
+  private void rewriteRestartPolicy(PodSpec podSpec, String podName, InternalEnvironment env) {
```

Comment:
Looks like tests are not modified in accordance with this change

Reply:
done